### DISCUSSION TITLE
Show total MW/GW electricity export and import for a country

### DISCUSSION THREAD

**Discussion by brmarconi** [electricitymaps/electricitymaps-contrib#4864] [Ideas]
It would be nice to show the total MW/GW electricity export and import for a country, as well as the relevant carbon intensity. Particularly show at a glance which part of the consumption of a country is covered by imports, and the contribution of those imports to the carbon intensity of the recipient country. It would help with the many discussions about countries sovereignty regarding electricity production and their responsibility with carbon emission.

I started giving it a try and modified the front end, here is what it could look like (look at Totals at the bottom):

![image](https://user-images.githubusercontent.com/120281919/207747242-f84a2e40-87eb-4bf0-8386-6943048d1ec3.png)

**Comment by VIKTORVAV99** [comment]
This is something that has been requested several times now and something I personally would like to see added.

With that said we are currently in a "feature freeze" for the frontend app as we are rewriting it as mentioned here: https://github.com/electricitymaps/electricitymaps-contrib/issues/4731

So while it is a great idea to add total net exchange in my own opinion it's not something we plan on doing right away. But after the refractor is done this is definitely something we can look at.

If you want to keep an eye on how the refractor is progressing you can check https://beta.electricitymaps.com. 🙂

**Reply by brmarconi** [reply]
Good move to TypeScript!
I'd suggest adding a warning along the lines you explained above in [Changes to the frontend](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Changes-to-the-frontend)
Actually it does say "make sure that it's based on an open issue that people agree with" but I wanted to illustrate my proposal :)

**Reply by VIKTORVAV99** [reply]
The plan is to get things done soon (hopefully) so I'm not sure we want to add it to the wiki just yet as we still accept bugfixes and geometry updates but I'll keep it in mind.

That last statment is more directed towards when people open PRs with new features that might not allign with the current plans or ideas.

**Comment by VIKTORVAV99** [comment]
Just linking a related request https://github.com/electricitymaps/electricitymaps-contrib/issues/4430
